dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -56,6 +56,12 @@ uniqueId.desc=The unique ID.
 authorizationEndpoint=Authorization end point URL
 authorizationEndpoint.desc=Specifies an Authorization end point URL.
 
+userInfoEndpoint=Userinfo end point URL
+userInfoEndpoint.desc=Specifies a UserInfo end point URL.
```

Comment:
I know the other endpoint descriptions don't have it, but should we provide more information about what the User Info endpoint is, what its format should be, what it's expected to return, etc.?

Reply:
perhaps, but paraphrasing the spec is a slippery slope.  Also most users will never see these fields, only those using WDT.